### DISCUSSION TITLE
How to create a bridge using layerzero

### DISCUSSION THREAD

**Discussion by harispericorn** [aptos-labs/aptos-developer-discussions#321] [Questions]
### Discord user ID

harikrishna2744

### Describe your question in detail.

I have implemented a bridge contracts from BCS testnet to Aptos testnet i have followed each steps of a sucessfull similar bridging transaction from which i have used the reference from my contract but the issue is when i sendToAptos the tx will go through in bcs but tx is getting inFlight forever
reference tx that i refereed https://snowtrace.io/address/0xA5972EeE0C9B5bBb89a5B16D1d65f94c9EF25166?sort=asc 

### What error, if any, are you getting?

`https://testnet.layerzeroscan.com/tx/0xab3f1d2df09c524c2183ccb8e949d71b5f900f955080094ad5bdc0c005ed4e99

this transaction is also stuck for bsc to aptos transfer 
TOKEN BRIDGE
BSC
0x6E0c0Cf0856Ad2D087344D630FFecB1C14800290
APTOS
27b30af21b6b1afcf6d62ffc54470d51d9d0deb52a4b6bbb44bcb953b77b714f`

### Which operating system are you using?

Linux (Ubuntu, Fedora, Windows WSL, etc.)

### Which SDK or tool are you using? (if any)

N/A

**Comment by 0x-j** [comment]
hi can you ask layerzero developers? sorry i don't have much context here
